### DISCUSSION TITLE
1px on top of the text in the input text, why?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#10977] [Help] [answered]
At this playground: https://play.tailwindcss.com/Q9jyQMAOuR

using this code:

```html
<div class="p-20">
  <input type="text" class="h-3.5 w-full rounded border border-gray-300 bg-gray-50 px-4 text-gray-950 outline-blue-600 focus:bg-white" value="HELLO!" />
</div>
```

I have this result:

![image](https://user-images.githubusercontent.com/41120635/230929001-5b4cfd81-8327-482b-bf26-d20fe1a983c7.png)

As you can see there is a (I think) 1px top on the text (between the text and the top border of the input.

Why?

**Comment by adamwathan** [comment, marked as answer]
That's just how fonts are made, each glyph is drawn within a bounding box and it's up to the font designer to decide how to use that space. It's very common for fonts to have a little space at the top to account for italics, as you can see here:

![image](https://user-images.githubusercontent.com/4323180/230939265-5d1f2730-018e-4640-8e0d-3b9e9a83248e.png)
